stable/mlrun-kit/templates/jupyter-notebook/pv.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.jupyterPersistencyStandalone.pvc.create -}}
```

Comment:
No need to add a new `jupyterPersistencyStandalone` block
You already have `jupyterNotebook.persistency.pvc.create` note that this file is creating a PV and not PVC, so should have a different flag for that

Reply:
pv,pvc  in separate block in order not to case  brake compatibility in original mlrun values.yaml